src/calendar/index.ts

```diff
@@ -121,11 +125,22 @@ export class CalendarBase<P extends CalendarProperties = CalendarProperties> ext
 		const {
 			month,
 			selectedDate = this._defaultDate,
-			year
+			year,
+			minDate,
+			maxDate
 		} = this.properties;
+
+		let selected = selectedDate;
```

Comment:
If I follow this right, it's going to set the `selected` date to the min/max if it's out of range rather than setting the component to invalid or not selecting anything. Is this the desired effect? Do we think this is a good user experience?

Reply:
We actually don't call this if the date is out of range so this can be removed.